### PR TITLE
chore: relocate surveys to home page

### PR DESCRIPTION
**Pull request by byronkimani**
- add surveys to home page
Screenshot_1651920927

Signed-off-by: Byron Kimani <byronkimani@gmail.com>

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -7,5 +7,5 @@ import 'package:myafyahub/domain/core/value_objects/app_context_constants.dart';
 import 'application/core/services/app_bootstrap.dart';
 
 Future<void> main() async {
-  await appBootStrap(testAppContexts);
+  await appBootStrap(demoAppContexts);
```
Comment (EugeneDevv) [inline lib/main.dart:10]: Revert this to testAppContext

**Review by EugeneDevv — CHANGES_REQUESTED** [review]

File: lib/application/core/services/utils.dart [hunk 2]
```diff
@@ -298,11 +298,6 @@ final List<UserProfileItemObj> userProfileItems = <UserProfileItemObj>[
     route: AppRoutes.clinicInformationPage,
     title: 'Clinic information',
   ),
-  UserProfileItemObj(
-    iconAssetPath: surveyItemImage,
-    route: AppRoutes.surveysPage,
-    title: 'Surveys',
-  ),
```
Comment (abiudrn) [inline lib/application/core/services/utils.dart:305]: This can still stay. There is no harm in having it here

File: lib/main.dart (hunk 1 above)
Comment (byronkimani) [reply lib/main.dart:10]: nice catch thanks!

**Review by abiudrn — APPROVED** [review]

File: lib/application/core/services/utils.dart (hunk 2 above)
Comment (byronkimani) [reply lib/application/core/services/utils.dart:305]: Noted

**Review by EugeneDevv — APPROVED** [review]

**Review by abiudrn — APPROVED** [review]

**Pull request merged**